lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -34,7 +34,7 @@ def serializable_hash_for_single_resource(options)
       def resource_relationships(options)
         relationships = {}
         serializer.associations(@include_tree).each do |association|
-          relationships[association.key] = relationship_value_for(association, options)
+          relationships[association.key] ||= relationship_value_for(association, options)
```

Comment:
Source of this should be investigated, that `serializer.associations(@include_tree).each` seems to yield each association multiple times